### ISSUE TITLE
v2.13 update message is not switching "Update" to "Download" for Windows builds

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5803]
And it should, because the update on Windows is broken and users need to Download and install the new version.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-4034)
┆Reporter: Beatriz Rizental Machado

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this while using the latest RC for the addons - https://archive.mozilla.org/pub/vpn/addons/candidates/20230130183600/ ( https://archive.mozilla.org/pub/vpn/addons/candidates/20230130183600/|smart-link ) - and the “*Download to Mozilla VPN 2.13*” is shown to 2.10, 2.11.0 and 2.12 Windows users → attaching a ss with the message.

!Screenshot 2023-01-31 at 13.50.14.png|width=1344,height=1121!

For the 2.11.1 version on Windows, on which the update works , and using the latest RC for addons, users are still seeing the “Download to Mozilla VPN 2.13” message instead of “*Update to Mozilla VPN 2.13*” → attaching a ss as well.

!image_2023_01_31T09_18_20_618Z.png|width=406,height=368!

Beatriz Rizental Machado / Santiago Andrigo, shouldn’t we show the “Update to Mozilla VPN 2.13” message to 2.11.1 windows users, instead of the Download one?

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

{quote}shouldn’t we show the “Update to Mozilla VPN 2.13” message to 2.11.1 windows users, instead of the Download one?{quote}

I honestly don’t know. Sebastian Streich ?

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

From Santiago Andrigo on Slack. 

{quote}My understanding is that the 'Update/Download' issue affects Windows clients 2.10, 2.11 and 2.12, but NOT 2.11.1, right?If so, the ideal is:{quote}

  * Windows users on versions <2.13 and not on version 2.11.1, get the 'Download 2.13' message.
  * Non-Windows users on versions <2.13 get the 'Update to 2.13' message.
  * In both cases, all other update / download messages are removed.

tl;dr; Yeah, the message should not show for 2.11 users. Putting this back to “In Progress”.
